samples/lspci.c

```diff
@@ -44,9 +44,10 @@ int main(void)
     const int bytes_per_line = 0x10;
     vfu_pci_hdr_id_t id = { 0 };
     vfu_pci_hdr_ss_t ss = { 0 };
-    vfu_pci_hdr_cc_t cc = { { 0 } };
+    vfu_pci_hdr_cc_t cc = { 0 };
```

Comment:
Looks like this is reverting the GCC4 build fix?

Reply:
Yes, sorry, will fix.